lib/models/os.coffee

```diff
@@ -54,8 +54,10 @@ getOsModel = (deps, opts) ->
 			{ versions, latest } = body
 			[ validVersions, invalidVersions ] = partition(versions, semver.valid)
 
-			validVersions.sort(osVersionRCompare)
```

Comment:
This no longer sorts the versions list, so the version list shown downstream won't be sorted at all. That definitely needs to stay I think.